boundary/collect.py

```diff
@@ -243,18 +243,29 @@ def mark_callback(op, caller):
 
     def collect_struct(self):
         public_fields = defaultdict(set)
+        parent_component_ref = {}
```

Comment:
the scope of parent_component_ref is stmt.walk_tree, why it appears here?

Reply:
This line doesn't change its scope or any functionality. I add it here just to improve readability.